cobalt/trace_event/json_file_outputter.cc

```diff
@@ -134,7 +135,7 @@ void JSONFileOutputter::Write(const char* buffer, int length) {
     return;
   }
 
-  int count = SbFileWrite(file_, buffer, length);
+  int count = starboard_api::FileWrite(file_, buffer, length);
```

Comment:
Consider routing this through `File::WriteAtCurrentPosNoBestEffort()` which does a SbFileWrite